### ISSUE TITLE
Logo file should be updated

### ISSUE THREAD

**Issue by cdw9** [boostorg/website-v2#1475]
Page examples:
* https://www.stage.boost.cppalliance.org/doc/libs/release/libs/assign/doc/index.html
* https://www.stage.boost.cppalliance.org/doc/libs/release/libs/compatibility/index.html

This logo is coming from https://www.boost.io/doc/libs/release/boost.png

<img src="https://github.com/user-attachments/assets/c226ea05-b9be-453d-95ae-cb613bafc092" width=500 />

**Comment by rbbeeston** [comment]
![Image](https://github.com/user-attachments/assets/91253a57-b4a7-4a13-9a2e-036ca5bbd2f4)

Replace with the above image. See if there is a single place where it's housed. if it's in multiple release libraries, just replace in 1.87 and develop libs

**Comment by rbbeeston** [comment]
yeah, no idea how many sources there cold be for it, but if you find it anywhere at the same size, go ahead and swap it.

If you find any at other sizes, point me to them and I'll generate replacements.

**Comment by cdw9** [comment]
The only other version I've found is https://github.com/boostorg/website/blob/master/boost.png - but it doesn't seem to be used within that repo, and I don't see it on the old site anywhere.

@rbbeeston https://github.com/boostorg/boost/pull/1012 still needs to be merged, but I don't have the option of adding reviewers on that commit.
